app/src/main/java/com/willowtree/vocable/keyboard/KeyboardViewModel.kt

```diff
@@ -16,23 +21,43 @@ class KeyboardViewModel : BaseViewModel() {
     }
 
     private val presetsRepository: PresetsRepository by inject()
+    private val mySayingsCategoryId: String =
+        get<Context>().getString(R.string.category_my_sayings_id)
 
     private val liveShowPhraseAdded = MutableLiveData<Boolean>()
     val showPhraseAdded: LiveData<Boolean> = liveShowPhraseAdded
 
     fun addNewPhrase(phraseStr: String) {
         backgroundScope.launch {
-            val categoryId = presetsRepository.getMySayingsId()
-            presetsRepository.addPhrase(
-                Phrase(
-                    System.currentTimeMillis(),
-                    System.currentTimeMillis(),
-                    true,
-                    0L,
-                    phraseStr,
-                    categoryId
+            val mySayingsCategory =
+                presetsRepository.getCategoryById(mySayingsCategoryId)
+            val phraseId = UUID.randomUUID().toString()
+            val mySayingsPhrases =
+                presetsRepository.getPhrasesForCategory(mySayingsCategory.categoryId)
+            with(presetsRepository) {
+                // TODO: Use currently set Locale
```

Comment:
Did you mean to do this before completing PR or is it still outstanding?

Reply:
this will be addressed in another PR